optaplanner-core/src/main/java/org/optaplanner/core/config/solver/SolverConfig.java

```diff
@@ -364,6 +367,14 @@ public void setSolutionClass(Class<?> solutionClass) {
         this.solutionClass = solutionClass;
     }
 
+    public DomainAccessType getDomainAccessType() {
+        return domainAccessType;
+    }
+
+    public void setDomainAccessType(DomainAccessType domainAccessType) {
+        this.domainAccessType = domainAccessType;
+    }
```

Comment:
We like the order of the "get/set" method combo to be in the order of the fields.
so both after setEntityClassList().

Motivation: It's just easier to see if a get/set/with method is lacking. Less chaotic.